galeracluster/source/installmysql.rst

```diff
@@ -177,3 +183,40 @@ Galera Cluster for MySQL is now installed on your server.  You need to repeat th
 .. note:: **See Also**: In the event that you installed Galera Cluster for MySQL over an existing standalone instance of MySQL, there are some additional steps that you need to take in order to update your system to the new database server.  For more information, see :doc:`migration`.
 
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+MySQL Shared Compatibility Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _`centos-mysql-shared-compt`:
+
+When installing Galera Cluster for MySQL on CentOS, versions 6 and 7, you may encounter a transaction check error that blocks the installation.  
+
+.. code-block:: text
+
+   Transaction Check Error:
+   file /usr/share/mysql/czech/errmsg.sys from install
+   mysql-wsrep-server-5.6-5.6.23-25.10.e16.x86_64 conflicts
+   with file from package mysql-libs-5.1.73-.3.e16_5.x86_64
+
+This relates dependency issue between the version of the MySQL shared compatibility libraries that CentOS uses and the one that Galera Cluster requires.  Upgrades are available through the Codership repository and you can install them with ``yum``.
+
+To install the compatibility libraries on CentOS, run the following command:
+
```

Comment:
The packages are named differently depending on CentOS version, so the commands for both distro versions are different. See original ticket https://github.com/codership/documentation/issues/132 .